pkg/tests/upgrade/test_salt_upgrade.py

```diff
@@ -28,6 +28,10 @@ def test_salt_upgrade(salt_call_cli, salt_minion, install_salt):
     assert ret.returncode == 0
     assert ret.data
 
+    # install dep following upgrade
```

Comment:
Can you add a note that states this will need to be removed when we come up with a solution to allow users to upgrade without re-installing pip dependencies.